examples/distributing/uniffi-rust-to-python-library/run.bat

```diff
@@ -0,0 +1,6 @@
+cargo run --features=uniffi/cli --bin uniffi-bindgen generate .\src\math.udl --language python --out-dir out
+python -m venv venv3
+call ".\venv3\Scripts\activate" 
+pip install yapf 
```

Comment:
I don't think this example needs `yapf`? And for this example I'm not sure a virtualenv is worthwhile?

Reply:
it may pass without yapf but it triggers a warning.

Reply:
could I keep it?